src/main/java/com/baidu/hugegraph/api/graphs/GraphsAPI.java

```diff
@@ -54,19 +81,30 @@ protected String type() {
         return result.readList(this.type(), String.class);
     }
 
+    public void clear(String graph, String message) {
+        String path = String.join("/", this.path(), graph, "clear");
```

Comment:
can add joinPath(path, graph, action) method: joinPath(this.path(), graph, "clear"),